packages/property-grid/package.json

```diff
@@ -54,7 +54,7 @@
     "@bentley/presentation-frontend": "2.3.3",
     "@bentley/product-settings-client": "2.3.3",
     "@bentley/ui-abstract": "2.3.3",
-    "@bentley/ui-components": "2.3.3",
+    "@bentley/ui-components": "2.9.2",
```

Comment:
you cant bump just one core pkg, its all or nothing. so please bump all the dev deps to 2.9.2, and the peer deps to ^2.9.2

Reply:
gotcha didn't realize I missed some, will do.

Reply:
Ok I definitely did this for the rest of the packages (just forgot the peer deps) and it didn't get committed, so I'm not totally useless. 👍  it's all pushed now